maths/runge_kutta_gills.py

```diff
@@ -0,0 +1,95 @@
+"""
+Use the Runge-Kutta-Gill's method of order 4 to solve Ordinary Differential Equations.
+
+https://en.wikipedia.org/wiki/Runge%E2%80%93Kutta_methods
+Author : Ravi Kumar
+"""
+from collections.abc import Callable
+from math import sqrt
+
+import numpy as np
+
+
+def runge_kutta_gills(
+    func: Callable,
```

Comment:
```suggestion
    func: Callable[[float, float], float],
```
Please annotate the parameter and output types